task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskFormDatePickers.vue

```diff
@@ -146,6 +153,26 @@ export default {
     };
   },
   computed: {
+    testToday() {
+      if (this.startDate > this.today){
+        return !this.isGreater;
+      } else {
+        return this.isGreater;}
+    },
+    testTomorrow() {
+      this.tomorrow.setDate(this.today.getDate() + 1);
+      if (this.startDate > this.tomorrow){
+        return !this.isGreater;
+      } else {
+        return this.isGreater;}
+    },
+    testNextWeek() {
+      this.nextWeek.setDate(this.today.getDate() + 7);
+      if (this.startDate > this.nextWeek){
+        return !this.isGreater;
+      } else {
+        return this.isGreater;}
+    },  
```

Comment:
You have a lot a repeated code. You should refactor to make it more readable, and with less repetition.
Take inspiration on function addBtnDate(), in which you can pass a parameter and allow you to set the date with only one function even if you call it for today, tomorrow or in one week)

Reply:
of course I have tested my changes, how would I commit a code without testing it & being sure of its rendering
here is a demo https://youtu.be/UgCsGKjb7gE

-- addBtnDate() it is not the best exemple to be ispired from in this case because it will do just one job which is to set the due date according to the user's input. However, my code will test the labels today / tomorrow / nextWeek one by one in order to check if the start date is greater than due date (the date may be greater than today and tomorrow but it is inferior to that of next week so here we should have 2 disabled buttons and only one that is enabled).

-- conditional statements verify if a specified condition is true so if already the first condition is approved the other ones will be ignored

Reply:
When I speak about inspiring addBtnDate, this is an example you can use to create a function
testDate(), that you can call with testDate(0) (for today), testDate(1) for tomorrow, or testDate(7) for next week
So that you have only one function which do the job, without repeating the code.

In addition, testDate is not a correct function name, it should be something more understandable.